x-pack/platform/test/spaces_api_integration/read_only_objects/apis/spaces/read_only_objects.ts

```diff
@@ -0,0 +1,521 @@
+/*
```

Comment:
I'm wondering if we can move this test under `x-pack/platform/test/spaces_api_integration/deployment_agnostic` so that it can run in all environments? We can also do this in a follow-up, @elena-shostak can definitely provide guidance here 🙂